src/languages/de-DE.js

```diff
@@ -215,7 +215,7 @@ export default {
     "Post URL": "Post URL",
     "Content Type": "Content Type",
     webhookJsonDesc: "{0} ist gut für alle modernen HTTP-Server sowie Express.js",
-    webhookFormDataDesc: "{multipart} ist gut für PHP. Die JSON muss mit {decodeFunction} geparst werden",
+    webhookFormDataDesc: "{multipart} ist gut für PHP. Die JSON muss mit {decodeFunction} verarbeitet werden",
```

Comment:
```suggestion
    webhookFormDataDesc: "{multipart} ist gut für PHP. Das JSON muss mit {decodeFunction} verarbeitet werden",
```
Das JSON-Objekt. If the JSON request (JSON-Anfrage) or JSON response (JSON-Antwort) is meant here, I would add that.